bin/test-workflow/2_admin_load_conjur_policies.sh

```diff
@@ -84,7 +86,7 @@ pushd policy > /dev/null
     sed "s#{{ TEST_APP_NAMESPACE_NAME }}#$TEST_APP_NAMESPACE_NAME#g" > ./generated/$TEST_APP_NAMESPACE_NAME.authn-any-policy-branch.yml
 popd > /dev/null
 
-set_namespace "$CONJUR_NAMESPACE"
+set_namespace "$CONJUR_NAMESPACE_NAME"
 
 announce "Finding or creating a Conjur CLI pod"
 conjur_cli_pod=$(get_conjur_cli_pod_name)
```

Comment:
I know these lines aren't part of your change, but `$(get_conjur_cli_pod_name)` should be double-quoted here and on line 96.